portiaui/app/templates/components/help-icon.hbs

```diff
@@ -2,6 +2,6 @@
     {{#if (eq tooltip.section 'tooltip')}}
         {{yield}}
     {{else}}
-        {{icon-button id=(concat "help-icon-" elementId) class="help-icon" icon='help'}}
+        {{icon-button id=(concat "help-icon-" elementId) class=classes icon=icon}}
```

Comment:
could we pass a single attribute in here that would set both classes and icon from a mapping?